src/ontology/edits/oeo-shared.omn

```diff
@@ -2494,6 +2494,33 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1390",
         OEO_00000014
     
     
+Class: OEO_00010361
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A renewable power unit is a power generating unit that has renewable energy as input.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue:https://github.com/OpenEnergyPlatform/ontology/issues/1423
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1437",
+        rdfs:label "renewable power unit"@en
+    
+    EquivalentTo: 
+        OEO_00000334
+         and (OEO_00010234 some OEO_00020085)
+    
+    SubClassOf: 
+        OEO_00000334
+    
+    
+Class: OEO_00010362
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A renewable power plant is a power plant that has renewable power units as parts.",
+        rdfs:label "renewable power plant"@en
+    
+    EquivalentTo: 
+        OEO_00000031
+         and (<http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00010361)
```

Comment:
Same question as above

Reply:
I don't see here any problem. The axiom here is just `renewable power plant EquivalentTo: 'power plant'  and ('has part' some 'renewable power unit')`, there is no mention of the energy input. So regarding the energy input, `renewable power plant` is fully axiomatised via `renewable power unit` and I don't see anything to adapt here.